tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -49,21 +49,27 @@ spec:
       --query cluster.resourcesVpcConfig.subnetIds --output text \
       )
 
-      nodes=$(params.desired-nodes)
+      nodes=$(($(params.desired-nodes)-1))
       asgs=$((nodes/1000))
       asg_name=$(params.cluster-name)-nodes
       create_dp()
       {
         CREATED_NODEGROUP=$(aws eks $ENDPOINT_FLAG --region $(params.region) list-nodegroups --cluster-name $(params.cluster-name)  --query 'nodegroups[?@==`'$asg_name-$1'`]' --output text)
-        #ToDo: paramterize instance-types
+        echo $3
+        EC2_INSTANCES=$3
```

Comment:
oh! by this comment [here](https://github.com/awslabs/kubernetes-iteration-toolkit/pull/325#discussion_r1036659368) I meant, adding as tekton param so we can dynamically alter it for pipeline/task runs as and when we get iced.

Reply:
okay..I will add this with prometheus taints PR.

Reply:
Have added this logic to a different comment.